editor/editor_node.cpp

```diff
@@ -6680,6 +6683,7 @@ int EditorNode::execute_and_show_output(const String &p_title, const String &p_p
 
 	eta.execute_output_thread.wait_to_finish();
 	execute_outputs->add_text("\nExit Code: " + itos(eta.exitcode));
+	print_line("\nExit Code: " + itos(eta.exitcode) + "\n");
```

Comment:
Use sentence case capitalization:

```suggestion
	print_line("\nExit code: " + itos(eta.exitcode) + "\n");
```